snuba/datasets/schemas/tables.py

```diff
@@ -41,12 +41,6 @@ def get_prewhere_candidates(self) -> Sequence[str]:
         return self.__prewhere_candidates
 
 
-class MigrationSchemaColumn(NamedTuple):
```

Comment:
It would have caused circular dependencies since we needed to use it in snuba/datasets/schemas/__init__.py (in the previous version that did not remove it completely)